filters/node.js

```diff
@@ -0,0 +1,29 @@
+import flatten from 'flat'; 
+
+export class NodeFilter {
+
+    async parseResource( resource ) {
+        const segments = [];
+        for(const [key, value] of Object.entries(flatten(resource))){
```

Comment:
can you put a space after `for`?

Reply:
Fixed.  Also added prettifier and ran eslint.

Reply:
i don't see them checked in and it's also not clear what rules you ran and whether they're consistent with the rest of the code. maybe it's better to add them in a separate PR, but the inconsistency is pretty obvious for example in the use of double quotes for strings when the rest of the code use single quotes.